src/lib.rs

```diff
@@ -98,6 +98,8 @@ pub struct Chart<'a> {
     x_label_format: LabelFormat,
     /// Y-axis label format.
     y_label_format: LabelFormat,
+    /// Y-axis tick label density
+    y_tick_labels: TickDisplay,
```

Comment:
Let's rename to `y_tick_display` for consistency?

Reply:
Done.